s3-spark/src/test/scala/geotrellis/spark/io/s3/util/S3RangeReaderProviderSpec.scala

```diff
@@ -0,0 +1,22 @@
+package geotrellis.spark.store.s3.util
+
+import geotrellis.store.s3.util._
+import geotrellis.spark.store.s3.testkit.MockS3Client
+import geotrellis.store.s3.S3ClientProducer
+import geotrellis.util.RangeReader
+
+import org.scalatest._
+
+class S3RangeReaderProviderSpec extends FunSpec with Matchers {
```

Comment:
Why this file is in `s3-spark` and not in `s3-store`? The implementation of the S3RangeReaderProvider is in `s3-store`.

Reply:
There wasn't a really great reason. We just have all of our tests for `s3-store` in `s3-spark` still, so I just put it there.

Reply:
@jbouffard can you look into the tests structure more?

Reply:
@pomadchin Actually, we can't move it to `s3-store` as it needs `TestUtils` which is in `s3-spark`.